src/GafferSceneTestModule/GafferSceneTestModule.cpp

```diff
@@ -44,19 +44,57 @@
 #include "GafferSceneTest/TestShader.h"
 #include "GafferSceneTest/TraverseScene.h"
 
+#include "GafferScene/Private/RendererAlgo.h"
+
 #include "GafferBindings/DependencyNodeBinding.h"
 
 #include "IECorePython/ScopedGILRelease.h"
 
 using namespace boost::python;
 using namespace GafferSceneTest;
 
+namespace
+{
+
 static void traverseSceneWrapper( const GafferScene::ScenePlug *scenePlug )
 {
 	IECorePython::ScopedGILRelease gilRelease;
 	traverseScene( scenePlug );
 }
 
+list testObjectSamples( const Gaffer::ObjectPlug &objectPlug, const std::vector<float> &sampleTimes, bool copy )
+{
+	std::vector<IECore::ConstObjectPtr> samples;
+	{
+		IECorePython::ScopedGILRelease gilRelease;
+		GafferScene::Private::RendererAlgo::objectSamples( &objectPlug, sampleTimes, samples );
+	}
+
+	list pythonSamples;
+	for( auto &s : samples )
+	{
+		if( copy )
+		{
+			pythonSamples.append( s->copy() );
+		}
+		else
+		{
+			pythonSamples.append( boost::const_pointer_cast<IECore::Object>( s ) );
+		}
+	}
+
+	return pythonSamples;
+}
+
+void testOutputCameras( const GafferScene::ScenePlug &scene, const IECore::CompoundObject &globals, const GafferScene::ScenePlug &renderSetsScene, IECoreScenePreview::Renderer &renderer )
+{
+	GafferScene::Private::RendererAlgo::RenderSets renderSets( &renderSetsScene );
+	IECorePython::ScopedGILRelease gilRelease;
```

Comment:
Switched to old bindings.  I've done the binding in src/GafferSceneModule/RenderBinding.cpp, since that was currently where we create GafferScene.Private, and that seemed like a relevant enough name.